{'description': {'en': 'Gentoo cross-compilation overlay for embedded devices'},
 'feed': ['https://cgit.gentoo.org/proj/embedded-cross.git/atom/'],
 'homepage': 'https://cgit.gentoo.org/proj/embedded-cross.git/',
 'name': 'embedded-cross',
 'owner': [{'email': 'armin76@gentoo.org',
            'name': u'Ra\xfal Porcel',
            'type': 'project'}],
 'quality': 'experimental',
 'source': [{'type': 'git',
             'uri': 'git://anongit.gentoo.org/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'https://anongit.gentoo.org/git/proj/embedded-cross.git'},
            {'type': 'git',
             'uri': 'git+ssh://git@git.gentoo.org/proj/embedded-cross.git'}],
 'status': 'official'}
$ pmaint sync embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at location '/home/mgorny/repos/eigenlay' lacks a defined repo_name
WARNING:pkgcore:repository at '/home/mgorny/repos/eigenlay', named 'eigenlay', doesn't specify masters in metadata/layout.conf. Defaulting to whatever repository is defined as 'default' (gentoo usually). Please explicitly set the masters, or set masters = '' if the repository is standalone.
Already up-to-date.
*** syncing embedded-cross
*** synced embedded-cross
 * Sync succeeded
$ pmaint regen -t 2 embedded-cross
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING:pkgcore:repository at '/home/mgorny/repos/steveeJ' has unsupported profile format(s): 'profile-set'
WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.062 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.063 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/e_dbus-0.5.0.061 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: app-editors/vim-core-7.2.442 failed.

!!! ERROR: app-editors/vim-core-7.2.303 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.


!!! ERROR: x11-libs/gtk+-2.14.5 failed.
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

WARNING: 'use_enable' function invoked by x11-libs/ewl-20090614 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
WARNING: 'use_enable' function invoked by x11-libs/ewl-20090729 during metadata regeneration.
WARNING: Please ensure upstream is aware of it and working to remove it.

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.303.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!        vim-core-7.2.303.ebuild, line   6:  Called inherit 'vim'

!!! ERROR: app-editors/vim-core-7.0.235 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.2.442.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '176' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        vim-core-7.2.442.ebuild, line   6:  Called inherit 'vim'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'

!!! ERROR: app-editors/vim-core-7.0.243 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: app-editors/vim-7.3 failed.
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed


!!! ERROR: x11-drivers/xf86-video-glamo-9999 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:

!!! ERROR: app-editors/vim-core-7.3 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-libs/gtk+/gtk+-2.14.5.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'

!!! ERROR: app-editors/vim-core-7.1.123 failed.

!!! ERROR: app-editors/vim-core-7.1.330 failed.
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             gtk+-2.14.5.ebuild, line   7:  Called inherit 'gnome.org' 'flag-o-matic' 'eutils' 'libtool' 'virtualx'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
 * 
 * "/home/mgorny/repos/embedded-cross/sci-geosciences/gpsd/gpsd-2.37.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
 * 
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'virtualx'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '228' 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '187' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.235.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'

!!! ERROR: x11-drivers/xf86-video-glamo-20100626 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        vim-core-7.0.235.ebuild, line   5:  Called inherit 'vim'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'

!!! ERROR: app-misc/mdbus-9999 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '160' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/virtualx.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!                 ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                virtualx.eclass, line  38:  Called die
!!! The die message:
!!!  virtualx eclass require EAPI=2 or newer.

!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '207' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.0.243.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        vim-core-7.0.243.ebuild, line   5:  Called inherit 'vim'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-9999.ebuild'
!!!                 ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '165' 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'

!!! ERROR: x11-apps/xdm-1.1.8 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!   xf86-video-glamo-9999.ebuild, line  10:  Called inherit 'x-modular'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.3.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                 ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '248' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.123.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim/vim-7.3.ebuild'
!!!                 ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!            vim-core-7.3.ebuild, line   7:  Called inherit 'vim'
!!!        vim-core-7.1.123.ebuild, line   5:  Called inherit 'vim'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!                 vim-7.3.ebuild, line   7:  Called inherit 'vim'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                         ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'

!!! ERROR: x11-wm/enlightenment-0.16.999.062 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-editors/vim-core/vim-core-7.1.330.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'

!!! ERROR: x11-terms/vala-terminal-1.2 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!              ebuild-daemon.lib, line  30:  Called __qa_invoke 'source' '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 179' for eclass vim

!!!                         ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
received unknown com: export CATEGORY='net-libs' PR=r0 PVR='2.8.6' PV='2.8.6' EBUILD='/home/mgorny/repos/embedded-cross/net-libs/gnutls/gnutls-2.8.6.ebuild' P='gnutls-2.8.6' PF='gnutls-2.8.6' PN=gnutlsfailed
Exception in thread Thread-12:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!        vim-core-7.1.330.ebuild, line   5:  Called inherit 'vim'

!!! ERROR: dev-lang/perl-5.10.1 failed.
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 184' for eclass vim

received unknown com: export CATEGORY='dev-util' PR=r0 PVR='4.5.18' PV='4.5.18' EBUILD='/home/mgorny/repos/embedded-cross/dev-util/strace/strace-4.5.18.ebuild' P='strace-4.5.18' PF='strace-4.5.18' PN=stracefailed
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-drivers/xf86-video-glamo/xf86-video-glamo-20100626.ebuild'
Exception in thread Thread-14:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '208' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/gentoo/eclass/x-modular.eclass'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '164' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'vim'
!!!                  ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!   xf86-video-glamo-20100626.ebuild, line  16:  Called inherit 'x-modular'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/app-misc/mdbus/mdbus-9999.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for vim failed

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'

!!! ERROR: sys-mobilephone/fso-raw-1.0.0 failed.
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!               x-modular.eclass, line  44:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '174' 'depend'
!!!                  ebuild-daemon.bash, line 210:  Called __ebd_main_loop

!!! ERROR: dev-lang/perl-5.8.8-r9 failed.
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
!!!                  ebuild-daemon.lib, line  23:  Called __qa_invoke '__preloaded_eclass_x-modular'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             isolated-functions.lib, line 286:  Called __preloaded_eclass_x-modular
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.2.ebuild'
 * 
 * "/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild":
 * Deprecation Warning: Usage of distutils.eclass in packages not supporting installation
 * for multiple Python ABIs in EAPI <=2 is deprecated.
 * The ebuild should to be fixed. Please report a bug, if it has not been already reported.
!!!              mdbus-9999.ebuild, line   5:  Called inherit 'git'
 * 
!!!       vala-terminal-1.2.ebuild, line   5:  Called inherit 'git' 'autotools'

!!! ERROR: sys-mobilephone/freesmartphone-framework-20090704 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-apps/xdm/xdm-1.1.8.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '200' 'depend'
!!!                 ebuild-daemon.bash, line 427:  Called inherit 'eutils' 'libtool' 'multilib' 'toolchain-funcs' 'flag-o-matic' 'autotools' 'git'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.10.1.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                  ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '245' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.10.1.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-apps/xdm/xdm-1.1.8.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.10.1.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                  ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             perl-5.10.1.ebuild, line   7:  Called inherit 'eutils' 'alternatives' 'flag-o-matic' 'toolchain-funcs' 'multilib'
!!!                         ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-apps/xdm/xdm-1.1.8.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.2.442
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.2.303
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.235
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.0.243
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: dev-util/strace-4.5.18
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: app-editors/vim-core-7.3
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: net-libs/gnutls-2.8.6
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.123
!!! caught exception inherit requires an unknown eclass, vim cannot be found while processing ebuild src: app-editors/vim-core-7.1.330
!!! caught exception Metadata Exception: pkg ebuild src: sci-geosciences/fso-gpsd-0.8, attr eapi
error: parsed eapi '0' doesn't match sourced eapi '2' while processing ebuild src: sci-geosciences/fso-gpsd-0.8
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-drivers/xf86-video-glamo-20100626
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: app-misc/mdbus-9999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-libs/gtk+-2.14.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/dialog-1.1.20080819
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/strace-4.5.15
!!! caught exception Metadata Exception: pkg ebuild src: x11-wm/enlightenment-0.16.999.062, attr eapi
error: parsed eapi '2' doesn't match sourced eapi '0' while processing ebuild src: x11-wm/enlightenment-0.16.999.062
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/perl-5.10.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/webkit-efl-0.1_p20090703
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.5
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-util/vala-dbus-binding-tool-0.1.6
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/freesmartphone-framework-20091021
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/perl-5.8.8-r9
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-raw-1.0.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/freesmartphone-framework-20090704
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-wireless/wmiconfig-9999
!!! caught exception inherit requires an unknown eclas!!!       ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'alternatives'

!!! ERROR: x11-terms/vala-terminal-1.1.1 failed.
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                         ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-raw/fso-raw-1.0.0.ebuild'
!!!               xdm-1.1.8.ebuild, line   8:  Called inherit 'multilib' 'x-modular' 'pam'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed


!!! ERROR: sys-mobilephone/apm2-2.0.0 failed.
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 285' for eclass alternatives

!!!                  ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.5.1' PV='0.5.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-0.5.1.ebuild' P='freesmartphone-framework-0.5.1' PF='freesmartphone-framework-0.5.1' PN='freesmartphone-framework'path

!!! ERROR: x11-base/xorg-server-1.5.3-r6 failed.
Exception in thread Thread-3:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!           fso-raw-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!                         ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '296' 'depend'

!!! ERROR: sys-mobilephone/fsomusicd-0.0.0 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! Call stack:

!!! ERROR: dev-lang/vala-0.8.1.1 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/fsogsmd-0.5.0 failed.
!!!                          ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.062.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '178' 'depend'

!!! ERROR: sys-mobilephone/fso-alsa-1.0.0 failed.
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                         ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                                 ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                          ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.062.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                                 ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!! The die message:
!!!  inherit for git failed

!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '208' 'depend'

!!! ERROR: sys-mobilephone/fsodatad-0.1.0 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '198' 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '218' 'depend'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 245' for eclass x-modular

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
received unknown com: export CATEGORY='x11-wm' PR=r0 PVR='0.16.999.061' PV='0.16.999.061' EBUILD='/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.061.ebuild' P='enlightenment-0.16.999.061' PF='enlightenment-0.16.999.061' PN=enlightenmentpath

!!! ERROR: sys-mobilephone/fso-abyss-0.9.1 failed.
Exception in thread Thread-4:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-wm/enlightenment/enlightenment-0.16.999.062.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '183' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.8.8-r9.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '215' 'depend'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '205' 'depend'
!!!   enlightenment-0.16.999.062.ebuild, line   8:  Called inherit 'enlightenment'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/freesmartphone-framework/freesmartphone-framework-20090704.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '179' 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.8.8-r9.ebuild'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsogsmd/fsogsmd-0.5.0.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!        ebuild-default-functions.lib, line 230:  Called __internal_inherit 'enlightenment'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!                   ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 201' for eclass enlightenment

received unknown com: export CATEGORY='x11-base' PR=r2 PVR='20081205-r2' PV=20081205 EBUILD='/home/mgorny/repos/embedded-cross/x11-base/xglamo/xglamo-20081205-r2.ebuild' P='xglamo-20081205' PF='xglamo-20081205-r2' PN=xglamopath
!!!   freesmartphone-framework-20090704.ebuild, line   6:  Called inherit 'git'
Exception in thread Thread-5:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '203' 'depend'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/x11-base/xorg-server/xorg-server-1.5.3-r6.ebuild'
!!!           fsogsmd-0.5.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-terms/vala-terminal/vala-terminal-1.1.1.ebuild'
!!!               ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!                          ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '210' 'depend'

!!! ERROR: sys-mobilephone/fsodeviced-0.9.4 failed.
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsomusicd/fsomusicd-0.0.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!     vala-terminal-1.1.1.ebuild, line   5:  Called inherit 'git' 'autotools'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/x11-base/xorg-server/xorg-server-1.5.3-r6.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 245' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1' PV='0.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-0.1.ebuild' P='libframeworkd-glib-0.1' PF='libframeworkd-glib-0.1' PN='libframeworkd-glib'path
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/perl/perl-5.8.8-r9.ebuild'
Exception in thread Thread-13:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-alsa/fso-alsa-1.0.0.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!           perl-5.8.8-r9.ebuild, line   5:  Called inherit 'eutils' 'alternatives' 'flag-o-matic' 'toolchain-funcs' 'multilib'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!         fsomusicd-0.0.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/x11-base/xorg-server/xorg-server-1.5.3-r6.ebuild'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!          fso-alsa-1.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop

!!! ERROR: sys-mobilephone/fsoinitd-0.1.0 failed.
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'alternatives'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '213' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!    xorg-server-1.5.3-r6.ebuild, line   8:  Called inherit 'x-modular' 'multilib'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/apm2/apm2-2.0.0.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 266' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090227 PV=20090227 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090227.ebuild' P='libframeworkd-glib-20090227' PF='libframeworkd-glib-20090227' PN='libframeworkd-glib'path
Exception in thread Thread-6:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 245' for eclass alternatives

!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='2010.05.11.2' PV='2010.05.11.2' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-specs/fso-specs-2010.05.11.2.ebuild' P='fso-specs-2010.05.11.2' PF='fso-specs-2010.05.11.2' PN='fso-specs'path
Exception in thread Thread-7:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodatad/fsodatad-0.1.0.ebuild'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'

!!! ERROR: sys-mobilephone/libframeworkd-glib-20090614 failed.
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'x-modular'
!!!              apm2-2.0.0.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!          fsodatad-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 198' for eclass x-modular

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.3.0' PV='0.3.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/fsotdld/fsotdld-0.3.0.ebuild' P='fsotdld-0.3.0' PF='fsotdld-0.3.0' PN=fsotdldpath
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/dev-lang/vala/vala-0.8.1.1.ebuild'
Exception in thread Thread-16:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                   ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '203' 'depend'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 266' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=99999999 PV=99999999 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-99999999.ebuild' P='libframeworkd-glib-99999999' PF='libframeworkd-glib-99999999' PN='libframeworkd-glib'path
s, git cannot be found while processing ebuild src: x11-terms/vala-terminal-1.1.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: net-libs/gnutls-2.8.5
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/apm2-2.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-base/xorg-server-1.5.3-r6
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/freesmartphone-framework-0.5.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: dev-lang/vala-0.8.1.1
!!! caught exception Metadata Exception: pkg ebuild src: sys-mobilephone/fsogsmd-0.5.0, attr eapi
error: parsed eapi '2' doesn't match sourced eapi '0' while processing ebuild src: sys-mobilephone/fsogsmd-0.5.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-alsa-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodatad-0.1.0
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/fso-abyss-0.9.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-wm/enlightenment-0.16.999.061
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: x11-base/xglamo-20081205-r2
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsodeviced-0.9.4
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-0.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsoinitd-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090227
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsoframework-0.8.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fso-specs-2010.05.11.2
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-20090614
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libfsotransport-0.9.6
!!! caught exception inherit requires an unknown eclass, git cannot be found while processing ebuild src: sys-mobilephone/libgsm0710-1.2.1
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsotdld-0.3.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-ui-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-glib-99999999
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/libphone-Exception in thread Thread-10:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!            vala-0.8.1.1.ebuild, line   7:  Called inherit 'gnome2'
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fso-abyss/fso-abyss-0.9.1.ebuild'

!!! ERROR: sys-mobilephone/fsonetworkd-0.1.0 failed.
!!!                   ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'gnome2'
!!!                   ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '266' 'depend'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsodeviced/fsodeviced-0.9.4.ebuild'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 228' for eclass gnome2

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.3.1' PV='0.9.3.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/gsm0710muxd/gsm0710muxd-0.9.3.1.ebuild' P='gsm0710muxd-0.9.3.1' PF='gsm0710muxd-0.9.3.1' PN=gsm0710muxdpath
Exception in thread Thread-1:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!         fso-abyss-0.9.1.ebuild, line   6:  Called inherit 'base' 'git' 'autotools'
!!!             ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!        fsodeviced-0.9.4.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                   ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!             ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!              ebuild-daemon.lib, line  35:  Called die
!!! The die message:
!!!  inherit for git failed

!!!                           ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'

!!! ERROR: sys-mobilephone/libframeworkd-phonegui-efl-20090506 failed.
!!!             ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '218' 'depend'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsoinitd/fsoinitd-0.1.0.ebuild'
!!!                           ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!! die invoked from directory /home/mgorny/pkgcore-venv/lib/pkgcore
!!! Call stack:
!!!             ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 286' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090724 PV=20090724 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui/libframeworkd-phonegui-20090724.ebuild' P='libframeworkd-phonegui-20090724' PF='libframeworkd-phonegui-20090724' PN='libframeworkd-phonegui'path
Exception in thread Thread-2:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                     ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!               isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-glib/libframeworkd-glib-20090614.ebuild'
!!!          fsoinitd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                           ebuild-daemon.bash, line 458:  Called __ebd_exec_main 'daemonize'
!!!                     ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!   libframeworkd-glib-20090614.ebuild, line   5:  Called inherit 'git' 'autotools'
!!!         isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/fsonetworkd/fsonetworkd-0.1.0.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!                           ebuild-daemon.bash, line 210:  Called __ebd_main_loop
!!!         ebuild-default-functions.lib, line 230:  Called __internal_inherit 'git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 223' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.9.9' PV='0.9.9' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libfsobasics/libfsobasics-0.9.9.ebuild' P='libfsobasics-0.9.9' PF='libfsobasics-0.9.9' PN=libfsobasicspath
Exception in thread Thread-9:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                    ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 213' for eclass git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='1.2.1' PV='1.2.1' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/libgsm0710/libgsm0710-1.2.1.ebuild' P='libgsm0710-1.2.1' PF='libgsm0710-1.2.1' PN=libgsm0710failed
!!!                           ebuild-daemon.bash, line 441:  Called __ebd_process_metadata '306' 'depend'
Exception in thread Thread-15:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!       fsonetworkd-0.1.0.ebuild, line   6:  Called inherit 'cornucopia_git'
!!!                           ebuild-daemon.bash, line 374:  Called __execute_phases 'depend'
!!!                                   ebuild.lib, line 431:  Called __load_ebuild '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090506.ebuild'
!!!   ebuild-default-functions.lib, line 230:  Called __internal_inherit 'cornucopia_git'
!!!              ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 193' for eclass cornucopia_git

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR='0.1.0' PV='0.1.0' EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/mioctl/mioctl-0.1.0.ebuild' P='mioctl-0.1.0' PF='mioctl-0.1.0' PN=mioctlpath
Exception in thread Thread-11:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

!!!                                   ebuild.lib, line 252:  Called __qa_invoke 'source' '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090506.ebuild'
!!!                       isolated-functions.lib, line 286:  Called source '/home/mgorny/repos/embedded-cross/sys-mobilephone/libframeworkd-phonegui-efl/libframeworkd-phonegui-efl-20090506.ebuild'
!!!   libframeworkd-phonegui-efl-20090506.ebuild, line  24:  Called inherit 'shr'
!!!                 ebuild-default-functions.lib, line 230:  Called __internal_inherit 'shr'
!!!                            ebuild-daemon.lib, line  37:  Called die
!!! The die message:
!!!  unknown inherit command from pythonic side, 'gen_metadata 224' for eclass shr

received unknown com: export CATEGORY='sys-mobilephone' PR=r0 PVR=20090506 PV=20090506 EBUILD='/home/mgorny/repos/embedded-cross/sys-mobilephone/ophonekitd/ophonekitd-20090506.ebuild' P='ophonekitd-20090506' PF='ophonekitd-20090506' PN=ophonekitdpath
Exception in thread Thread-8:
Traceback (most recent call last):
  File "/usr/lib64/python2.7/threading.py", line 810, in __bootstrap_inner
    self.run()
  File "/usr/lib64/python2.7/threading.py", line 763, in run
    self.__target(*self.__args, **self.__kwargs)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/operations/regen.py", line 15, in regen_iter
    regen_func(x)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/repository.py", line 521, in __call__
    return pkg._fetch_metadata(ebp=self.ebp, force_regen=self.force)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 269, in _fetch_metadata
    return self._parent._get_metadata(self, ebp=ebp, force_regen=force_regen)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 360, in _get_metadata
    return self._update_metadata(pkg, ebp=ebp)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/ebuild_src.py", line 368, in _update_metadata
    mydata = my_proc.get_keys(pkg, self._ecache)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 827, in get_keys
    {"key": receive_key})
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 760, in _run_depend_like_phase
    self.write("%s %i\n%s" % (command, len(data), data), append_newline=False)
  File "/home/mgorny/pkgcore-venv/lib/python2.7/site-packages/pkgcore/ebuild/processor.py", line 458, in write
    raise RuntimeError(ie)
RuntimeError: [Errno 32] Broken pipe

utils-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/fsonetworkd-0.1.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/gsm0710muxd-0.9.3.1
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/mkdump-1.0.0
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-efl-20090506
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libframeworkd-phonegui-20090724
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-20090614
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libfsobasics-0.9.9
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/ophonekitd-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/libphone-ui-shr-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/mioctl-0.1.0
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/phoneuid-99999999
ERROR pkgcore: returned val from gen_metadata was 'False'
!!! caught exception False while processing ebuild src: sys-mobilephone/ophonekitd-20090506
!!! caught exception ("Expected command; instead got nothing from ''", None) while processing ebuild src: sys-mobilephone/pyphonelog-99999999
Error running an operation:
 Unhandled exception in regen_cache
 * Cache regen failed with 246
